### PR TITLE
Use 'server:start' instead of 'server:run' when possible

### PR DESCRIPTION
**Pull request by javiereguiluz**
`server:start` is a better option ... but it's not always possible to run it.

### CODE REVIEW

**Comment by lyrixx** [comment]
Should be deal with Symfony Version also ? because this command does not exist in older SF version.

**Comment by javiereguiluz** [comment]
@lyrixx you are right! I've improved the condition to check that we're using Sf 2.6.0 or higher. Thanks!

File: src/Symfony/Installer/DemoCommand.php [hunk 1]
```diff
@@ -145,11 +145,13 @@ private function displayInstallationResult()
             ));
         }
 
+        $serverRunCommand = version_compare($this->version, '2.6.0', '>=') && extension_loaded('pcntl') ? 'server:start' : 'server:run';
```
Comment (xabbuh) [inline src/Symfony/Installer/DemoCommand.php]: Do we need this for the demo application?

File: src/Symfony/Installer/NewCommand.php [hunk 2]
```diff
@@ -256,14 +256,15 @@ protected function displayInstallationResult()
         }
 
         $consoleDir = ($this->isSymfony3() ? 'bin' : 'app');
+        $serverRunCommand = version_compare($this->version, '2.6.0', '>=') && extension_loaded('pcntl') ? 'server:start' : 'server:run';
```
Comment (xabbuh) [inline src/Symfony/Installer/NewCommand.php]: You also need to check for HHVM.

File: src/Symfony/Installer/DemoCommand.php (hunk 1 above)
Comment (javiereguiluz) [inline src/Symfony/Installer/DemoCommand.php]: I don't know why should it be different for the demo app 😕

Comment (xabbuh) [inline src/Symfony/Installer/DemoCommand.php]: You cannot select a version of the demo application, can you? And the one that is going to be installed is using Symfony 3 already, isn't it?

Comment (javiereguiluz) [inline src/Symfony/Installer/DemoCommand.php]: You are so right!! I'm sorry for this mistake. It's now fixed. Thanks!

**Pull request closed without merging**